AirCasting/ABConnector/AirBeam3Configurator.swift

```diff
@@ -11,7 +11,7 @@ import Resolver
 
 protocol AirBeamConfigurator {
     func configureMobileSession(location: CLLocationCoordinate2D, completion: @escaping (Result<Void, Error>) -> Void)
-    func configureFixed(uuid: SessionUUID, completion: @escaping (Result<Void, Error>) -> Void)
+    func configureSession(uuid: SessionUUID, completion: @escaping (Result<Void, Error>) -> Void)
     func configureFixedCellularSession(uuid: SessionUUID,
```

Comment:
side note: this protocol is also a good candidate for a small refactor so it uses proper polymorphism - we'd have a single method here like `configureSession(completion:)` and then different implementations for fixed/mobile/cellular/whatever. This way we'd make one step towards unification of the session creation process